src/components/Cars.jsx

```diff
@@ -0,0 +1,36 @@
+import cn from 'classnames';
+
+const Cars = ({ cars, selectedCarId, setSelectedCarId }) => (
+  <table>
+    <thead>
+      <tr>
+        <th>Id</th>
+        <th>Brand</th>
+        <th>Color</th>
+        <th>Rent price</th>
+      </tr>
+    </thead>
+    <tbody>
+      {cars.map(car => (
+        <tr
+          key={car.id}
+          className={cn({
+            'is-active': car.id === selectedCarId,
+          })}
+          onClick={() => {
+            setSelectedCarId(car.id);
+          }}
```

Comment:
Selecting the car doesn't work correctly. Also, it is not required by the task description